### PR TITLE
Use transport worker

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

File: src/proxy/index.ts [hunk 1]
```diff
@@ -93,13 +93,20 @@ export default class Proxy extends Router {
 
     _registerServiceRoutes (developmentMode: boolean) {
         const hammerheadFileName      = developmentMode ? 'hammerhead.js' : 'hammerhead.min.js';
-        const hammerheadScriptContent = <Buffer>read(`../client/${hammerheadFileName}`);
+        const hammerheadScriptContent = read(`../client/${hammerheadFileName}`) as Buffer;
+        const transportWorkerContent  = read('../client/transport-worker.js') as Buffer;
```
Comment (miherlosev) [inline src/proxy/index.ts]: Should be same as for `hammerheardFileName`:
```js
const transportWorkerFileName = developmentMode ? 'transport-worker.js` : `transport-worker.min.js`;
```

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

File: Gulpfile.js [hunk 2]
```diff
@@ -162,7 +167,11 @@ gulp.step('client-scripts-processing', () => {
         .pipe(uglify())
         .pipe(rename('hammerhead.min.js'));
 
-    return mergeStreams(script, bundledScript)
+    const uglifiedTransportWorker = gulp.src('./lib/client/transport-worker.js')
```
Comment (miherlosev) [inline Gulpfile.js]: `const bundledTransportWorker = ...`

File: src/proxy/index.ts [hunk 3]
```diff
@@ -92,12 +92,19 @@ export default class Proxy extends Router {
     _registerServiceRoutes (developmentMode: boolean) {
         const hammerheadFileName      = developmentMode ? 'hammerhead.js' : 'hammerhead.min.js';
         const hammerheadScriptContent = read(`../client/${hammerheadFileName}`) as Buffer;
+        const transportWorkerFileName = developmentMode ? 'transport-worker.js' : 'transport-worker.min.js';
```
Comment (miherlosev) [inline src/proxy/index.ts]:
```js
const hammerheadFileName     = 'hammerhead' + this._getExtention (developmentMode);
const transportWorkerFileName = 'transport-worker.js` + this.getExtention(developmentMode);
```

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Pull request merged**